templates/inspec/plural_resource.erb

```diff
@@ -51,7 +51,7 @@ base = "'#{object.self_link_url[0].join}'"
   end
 
   def fetch_resource(params)
-    get_request = inspec.backend.fetch(base, url, params)
+    inspec.backend.fetch_all(base, url, params)
```

Comment:
Where does fetch_all relate to fetch_wrapped_resource?

Reply:
fetch_wrapped_resource calls fetch_resource which calls fetch_all on the InSpec train plugin which returns an array that has all of the responses (if pagination was needed).

Then fetch_wrapped_resource unwraps each of the responses using the 'items' key and converts them into an array of hashes that are symbol -> object that represent each resource. These are aggregated by InSpec's FilterTable that basically knows how to handle json objects.